src/pc/rom_checker.cpp

```diff
@@ -1,8 +1,24 @@
 #include <fstream>
 #include <iostream>
 #include <vector>
-#include <filesystem>
+
+#ifndef __has_include
```

Comment:
To summarize some of the discussion briefly, in order for the change in this file to have a useful effect, it would need to be combined and tested at the same time with analogous changes (non-PPA Ubuntu 20.04 compatibility)  in these other repos

https://github.com/djoslin0/coopnet
https://github.com/paullouisageneau/libjuice
https://github.com/lua/lua/tree/063d4e4543088e7a21965bda8ee5a0f952a9029e

Alternatively, the precompiled build can be run in Ubuntu 20.04, or a build of unmodified source code can be made to build and run in Ubuntu 20.04, by using the commands to add the PPA, so that is an easier way that could fit in a documentation file without editing any code.

I'm going to make and test a version that works on Ubuntu 20.04 without a PPA eventually but I'm not at that part yet.